### ISSUE TITLE
short inventory list

### ISSUE THREAD

**Issue by rodrigoaguilar11** [FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher#75]
When playing in 1024x768 resolution with or without a patch, the inventory list is longer.
but any other resolution with or without patch keeps it short of only 5 items in length, it should adapt like the purchase menus or maybe stay as 1024x768.

Inventory in 1024x768:
![04690ea5-df53-44a4-8282-c0f20d36f968](https://github.com/FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher/assets/110432593/2a92f1c3-54d0-435b-a219-f9cac5d46899)

Inventory in 1280x720
![84accfc0-2b55-4574-a5de-8ff5d6ea40cf](https://github.com/FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher/assets/110432593/3f75105b-be2b-4db2-929d-c83c51ced36d)

Purchase menus:
![c3396068-9686-42e3-af94-fcbbbe11571b](https://github.com/FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher/assets/110432593/1f95c322-73ae-4e84-98cf-079e4a3b3492)

**Comment by FaithBeam** [comment]
Thank you for reporting this. This behavior must be controlled by the two default resolutions of the game, 800x600 and 1024x768. The patcher only edits the 800x600 resolution so it only shows a max of 5 items in the inventory. I can probably edit the 1024x768 resolution to the custom one which hopefully will fix this.

**Comment by FaithBeam** [comment]
Well, I was wrong or partially right. I edited the 1024x768 bytes to the custom resolution and ran the shortcut with the -r1024x768 option, and it still shows 5 items. Uninstalling the patch and running the same shortcut shows more items. Maybe there's some other resolution bytes I need to edit?

**Comment by rodrigoaguilar11** [comment]
> Well, I was wrong or partially right. I edited the 1024x768 bytes to the custom resolution and ran the shortcut with the -r1024x768 option, and it still shows 5 items. Uninstalling the patch and running the same shortcut shows more items. Maybe there's some other resolution bytes I need to edit?

yes , according to my experience, patching the game on 1024x768 extends the inventory list to 9 objects, I understand that the default executable does it on 800x600, I guess running it on 1024x768 is a condition to extend the inventory, That condition should be modified to a custom resolution I suppose.
Or you can change the list size.
It is curious that the list so build or buy automatically adapts to the resolution but this not.
